openshift_metrics/config.py

```diff
@@ -14,3 +14,21 @@
 S3_SECRET_ACCESS_KEY = os.getenv("S3_OUTPUT_SECRET_ACCESS_KEY")
 S3_INVOICE_BUCKET = os.getenv("S3_INVOICE_BUCKET", "nerc-invoicing")
 S3_METRICS_BUCKET = os.getenv("S3_METRICS_BUCKET", "openshift_metrics")
+
+# Billing Configuration
+# Service Unit Names for Rate Lookup from nerc-rates
+SU_NAMES = ["GPUV100", "GPUA100", "GPUA100SXM4", "GPUH100", "CPU"]
+RESOURCE_NAMES = ["vCPUs", "RAM", "GPUs"]
+
+# Rate Configuration
+USE_NERC_RATES = os.getenv("USE_NERC_RATES", "false").lower() == "true"
+
+# Individual rates (used when USE_NERC_RATES=false)
+RATE_CPU_SU = os.getenv("RATE_CPU_SU", "0.013")
+RATE_GPU_V100_SU = os.getenv("RATE_GPU_V100_SU", "1.214")
+RATE_GPU_A100SXM4_SU = os.getenv("RATE_GPU_A100SXM4_SU", "2.078")
+RATE_GPU_A100_SU = os.getenv("RATE_GPU_A100_SU", "1.803")
+RATE_GPU_H100_SU = os.getenv("RATE_GPU_H100_SU", "6.04")
```

Comment:
@naved001 Do we still want keep the old behavior of having the rates `None` if not provided?